aiven/client/client.py

```diff
@@ -1217,13 +1248,13 @@ def request_project_vpc_peering_connection(self, project, project_vpc_id, peer_c
 
     def delete_project_vpc_peering_connection(
         self,
-        project,
-        project_vpc_id,
-        peer_cloud_account,
-        peer_vpc,
-        peer_region=None,
-        peer_resource_group=UNDEFINED,
-    ):
+        project: str,
+        project_vpc_id: str,
+        peer_cloud_account: str,
+        peer_vpc: str,
+        peer_region: Optional[str] = None,
+        peer_resource_group: Any = UNDEFINED,
```

Comment:
`Any` -> `Union[object, str]`
Makes no difference for mypy, but preserves an info about `str`.